Week-2/CSS Advanced/Receipe with flex/oatmeal_cookies.html

```diff
@@ -0,0 +1,94 @@
+<!DOCTYPE html>
+<html lang="en">
+   <head>
+      <meta charset="UTF-8">
+      <title>binary eats - Oatmeal Coconut Pantry Cookies</title>
+      <link href="styles.css" rel="stylesheet">
+   </head>
+
+   <body>
+      <header>
+         <div class="logo">binary eats</div>
+         <div>
+            <a href="">sign-in</a>
+            <a href="">search</a>
+         </div>
+      </header>
+      <div class="content">
+         <div class="main-content">
+            <div class="title">
+               <h1>OATMEAL COCONUT PANTRY COOKIES</h1>
+               <p>Chef Roman, August 24, 2018</p>
+            </div>
+            <div class="summary">
+               <img src="oatmealCookie_small.jpg" alt="Photo of two of the world's best oatmeal cookies">
+               <div>
+                  <div>
+                     <p>Prep Time:</p>
+                     <p>30 minutes</p>
+                  </div>
+                  <div>
+                     <p>Cooking Time:</p>
+                     <p>9-13 minutes</p>
+                  </div>
+                  <div>
+                     <p>Skill level:</p>
+                     <p>Easy</p>
+                  </div>
+                  <div>
+                     <p>Makes:</p>
+                     <p>24 cookies</p>
+                  </div>
+               </div>
+            </div>
+            <div class="ingredients">
+               <h2>Ingredients</h2>
```

Comment:
Learning level comment. Instaed of hard coded li. If you have added all the li values in array and rendered them from for loop using js. It would be great. Nothing wrong done, just a comment!